musi/pk.srt

```diff
@@ -2595,207 +2595,271 @@ please.
 01:12:07,877 --> 01:12:09,468
-first get me my remote.
+nanpa wan la o pana e ilo lawa mi.
+==first get me my remote.
 
 602
 01:12:09,548 --> 01:12:11,118
-which remote?
+ilo lawa seme?
+==which remote?
 
 603
 01:12:11,118 --> 01:12:12,604
-Spaceship's.
+pi tomo tawa mun.
+==Spaceship's.
 
 604
 01:12:13,437 --> 01:12:14,191
-spaceship?
+tomo tawa mun?
+==spaceship?
 
 605
 01:12:14,350 --> 01:12:18,190
-Forgotton? I told you so many times.
+sina weka ala weka? tenpo mute la mi toki e ijo ona.
+==Forgotton? I told you so many times.
 
 606
 01:12:18,477 --> 01:12:22,290
-I am not from this planet,
+mi tan ma ni ala. mi tan ma pi poka ala.
+==I am not from this planet,
 came from far away.
 
 607
 01:12:22,953 --> 01:12:26,564
-Just get me my remote and i'll go home.
+sina pana e ilo lawa tawa mi la mi tawa tomo mi.
+==Just get me my remote and i'll go home.
 
 608
 01:12:27,080 --> 01:12:29,937
-brother let me go,
+o weka e mi tan tomo ni.
+mi mama pi jan lili.
+==brother let me go,
 I have small children.
 
 609
 01:12:30,374 --> 01:12:31,302
-i know it very well.
+mi sona pona e ni.
+==i know it very well.
 
 610
 01:12:31,747 --> 01:12:34,477
-Ganesh and kartik. They aren't little and can survive alone.
+sewi Kanesi en sewi Kateki. ona li lili ala li ken lon poka ala sina.
+==Ganesh and kartik. They aren't little and can survive alone.
 
 611
 01:12:34,477 --> 01:12:38,955
-you get me my remote and then
+sina pana e ilo lawa mi la sina ken
+tawa kulupu mama sina.
+==you get me my remote and then
 go to your family.
 
 612
 01:12:39,239 --> 01:12:40,906
-Oh god save me.
+sewi o weka e mi tan ni.
+==Oh god save me.
 
 613
 01:12:40,985 --> 01:12:43,289
-there is a God above you?
+sewi ante li sewi tawa sina anu seme?
+==there is a God above you?
 
 614
 01:12:43,289 --> 01:12:44,455
-Om namah shivaya
+[toki sewi Palata]
+==Om namah shivaya
 
 615
 01:12:44,636 --> 01:12:45,971
-where is he?
+ona li lon seme?
+==where is he?
 
 616
 01:12:51,699 --> 01:12:53,309
-Save me!
+o weka e mi tan ni!
+==Save me!
 
 617
 01:13:02,255 --> 01:13:07,151
-Somebody help!
+jan o pana e pona!
+==Somebody help!
 
 618
 01:13:52,096 --> 01:13:58,755
-I was in the himalaya..meditating
+mi lon nena Imalaja... li pilin e sewi.
+==I was in the himalaya..meditating
 
 619
 01:13:59,191 --> 01:14:11,298
-..then suddenly i see light,
+ni la suno li kama
+tan ijo lili ni lon anpa.
+==..then suddenly i see light,
 from this small object on the ground.
 
 620
 01:14:11,493 --> 01:14:19,236
-I closed my eyes thought about God,
+mi pini e oko mi li pilin e sewi.
+sewi li suno e mi.
+==I closed my eyes thought about God,
 and i got enlighted.
 
 621
 01:14:19,509 --> 01:14:27,952
-this object was so hot even in
+ijo ni li seli mute. taso ona
+li lon lete mute.
+==this object was so hot even in
 a very cold weather.
 
 622
 01:14:28,080 --> 01:14:35,795
-God confirmed its the broken piece of
+sewi li toki e ni: ijo ni li wan tan
+ilo kalama pi lawa Siwa. o kama jo e ona.

```

Comment:
"lawa" is the head, but not the leader. (This happens repeatedly in this scene.)
lawa → jan lawa